plutus-core/plutus-core/test/Pretty/Readable.hs

```diff
@@ -31,8 +31,7 @@ test_PrettyReadable =
   where
     folder :: Pretty fun => PlcFolderContents DefaultUni fun -> TestTree
     folder
-        = runTestNestedIn ["plutus-core", "test", "Pretty", "Golden"]
-        . testNested "Readable"
+        = runTestNested ["plutus-core", "test", "Pretty", "Golden", "Readable"]
```

Comment:
I've realized that we do `testNested` after every call to `runTestNestedIn`, so we could as well just incorporate the former into the latter and save on typing and complexity.